client/ionic/src/App.tsx

```diff
@@ -46,11 +46,6 @@ const App: React.FC = () => (
           component={SplashCarousel}
           exact={true}
         />
-        <Route
```

Comment:
Do you want to remove the CheckYourHealth component file / yml file entirely?

Reply:
I'm going to leave it for now just because the content is still a bit of a question mark. We can do some cleanup post v0.1 👍